app/main.ipynb

```diff
@@ -151,17 +188,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "df[\"week_day\"] = df[\"datetime\"].dt.dayofweek\n",
+    "day_sales = df.groupby(\"week_day\")[\"item_price\"].sum()\n",
+    "day_names = [\"Monday\", \"Tuesday\", \"Wednesday\", \"Thursday\", \"Friday\", \"Saturday\", \"Sunday\"]\n",
+    "day_sales.index = day_names\n",
```

Comment:
The assignment of `day_names` to `day_sales.index` is incorrect. `day_sales.index` should be mapped to `day_names` using the `map` function or similar to ensure the correct day names are assigned.